Implementation/core/Cafeteria.py

```diff
@@ -0,0 +1,70 @@
+from data.Database import connect_to_db
+from datetime import datetime
+from utils.MenuSorter import MenuSorter
+
+class MenuItem:
+    def __init__(self, menu_item_id, menu_item_name, price, availability, food_type, spice_level, is_sweet):
+        self.menu_item_id = menu_item_id
+        self.menu_item_name = menu_item_name
+        self.price = price
+        self.availability = availability
+        self.food_type = food_type
+        self.spice_level = spice_level
+        self.is_sweet = is_sweet
+
+class Feedback:
+    def __init__(self, feedback_id, menu_item_id, user_id, comment, rating, feedback_date):
+        self.feedback_id = feedback_id
+        self.menu_item_id = menu_item_id
+        self.user_id = user_id
+        self.comment = comment
+        self.rating = rating
+        self.feedback_date = feedback_date
+
+class Recommendation:
+    def __init__(self, recommendation_id, menu_item_id, date, meal_type):
+        self.recommendation_id = recommendation_id
+        self.menu_item_id = menu_item_id
+        self.date = date
+        self.meal_type = meal_type
+
+class UserPreference:
+    def __init__(self, preference_id, user_id, menu_item_id, preference_date=None):
+        self.preference_id = preference_id
+        self.user_id = user_id
+        self.menu_item_id = menu_item_id
+        self.preference_date = preference_date if preference_date else datetime.now()
+
+class Menu:
+    def display_menu_items(self):
+        conn = connect_to_db()
+        cursor = conn.cursor()
+        cursor.execute("SELECT menu_item_id, menu_item_name, price FROM MenuItems WHERE availability = '1'")
+        food_items = cursor.fetchall()
+        conn.close()
+
+        display = f"{'Item ID':<10} {'Food Item':<20} {'Price':>10}\n" + "-" * 40 + "\n"
+        for food_item in food_items:
+            display += f"{food_item[0]:<10} {food_item[1]:<20} {food_item[2]:>10.2f}\n"
+        return display
+
+    def display_recommended_menu(self, preferences=None):
```

Comment:
Functions like these do not seem to belong to Menu class, which by name seems like a model class.